web/src/main/java/co/mobileaction/example/web/repository/IPostRepository.java

```diff
@@ -13,4 +14,7 @@
 public interface IPostRepository extends JpaRepository<Post, Long>
 {
     List<Post> findAllByUserId(Long userId);
+
+    @Query("SELECT DISTINCT p.userId FROM Post p")
+    List<Long> findDistinctUsers();

```

Comment:
we can use Set to let other services know that this returns distinct user ids
```suggestion
    Set<Long> findDistinctUserIds();

```